app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :payments
+
+  # validates :name, presence: true, length: { minimum: 3 }
+  # validates :user_id, presence: true, numericality: { greater_than: 0 }
```

Comment:
- [ ] Let's remove commented code from your project. This makes it hard to read it.

Reply:
Implemented thank you for reviewing. :+1: